### PR TITLE
ON-1240 ILR data fields for data accuracy

### PR DESCRIPTION
**Pull request by arshedmust**

### CODE REVIEW

**Review by alandburns — APPROVED** [review]

File: src/SharedPipelineComponents/IlrSubmissionEvents/SFA.DAS.Provider.Events.Submission/SubmissionEventsProcessor.cs [hunk 1]
```diff
@@ -109,16 +109,6 @@ private SubmissionEvent CalculateDelta(IlrDetails currentIlr, IlrDetails lastSee
                 (@event = @event ?? new SubmissionEvent()).PathwayCode = currentIlr.PathwayCode;
             }
 
-            if (currentIlr.ActualStartDate != lastSeenIlr?.ActualStartDate)
```
Comment (carlbm) [inline src/SharedPipelineComponents/IlrSubmissionEvents/SFA.DAS.Provider.Events.Submission/SubmissionEventsProcessor.cs]: Why was this removed?

**Review by carlbm — CHANGES_REQUESTED** [review]
Curious about the removed logic

File: src/SharedPipelineComponents/IlrSubmissionEvents/SFA.DAS.Provider.Events.Submission/SubmissionEventsProcessor.cs (hunk 1 above)
Comment (alandburns) [reply src/SharedPipelineComponents/IlrSubmissionEvents/SFA.DAS.Provider.Events.Submission/SubmissionEventsProcessor.cs]: I believe it is because they are being handled at line 139-140 instead,
                || currentIlr.ActualStartDate != lastSeenIlr?.ActualStartDate
                || currentIlr.PlannedEndDate != lastSeenIlr?.PlannedEndDate
but if this is likely to be an issue elsewhere the original lines 112 - 121 can remain.

Comment (alandburns) [reply src/SharedPipelineComponents/IlrSubmissionEvents/SFA.DAS.Provider.Events.Submission/SubmissionEventsProcessor.cs]: As there is no impact, this will be reverted

**Comment by alandburns** [comment]
Closing this branch as we will not be implementing the changes to Submission Events and Provider Events with the closing of Payments V1. And Payments V2 will not be supporting the data we need.
